kbs/src/token/mod.rs

```diff
@@ -16,7 +16,7 @@ pub const TOKEN_TEE_PUBKEY_PATH_ITA: &str = "/attester_runtime_data/tee-pubkey";
 pub const TOKEN_TEE_PUBKEY_PATH_ITA_VTPM: &str = "/attester_user_data/tee-pubkey";
 pub const TOKEN_TEE_PUBKEY_PATH_COCO: &str = "/customized_claims/runtime_data/tee-pubkey";
 pub const TOKEN_TEE_PUBKEY_PATH_EAR: &str =
-    "/submods/cpu/ear.veraison.annotated-evidence/runtime_data_claims/tee-pubkey";
+    "/submods/cpu0/ear.veraison.annotated-evidence/runtime_data_claims/tee-pubkey";
```

Comment:
we can move this into the [extension](https://www.ietf.org/archive/id/draft-fv-rats-ear-05.html#name-ear-extensions) field to get rid of depending on hardcode `cpu0` name and also ear [crate](https://docs.rs/ear/0.3.0/ear/struct.Extensions.html) supports this well.  I think it it the good time to get this done in this PR. wdyt?

Reply:
i'll look into this once I get everything fixed up. might be one for a follow-up